### PR TITLE
changed bing search to use bing dev search

### PR DESCRIPTION
**Pull request by heejaechang**

### CODE REVIEW

File: src/EditorFeatures/Core/Implementation/Suggestions/SuggestedAction.cs [hunk 1]
```diff
@@ -163,6 +164,9 @@ public virtual bool HasPreview
             // Light bulb will always invoke this function on the UI thread.
             AssertIsForeground();
 
+            var preferredDocumentId = Workspace.GetDocumentIdInCurrentContext(SubjectBuffer.AsTextContainer());
+            var preferredProjectid = preferredDocumentId == null ? null : preferredDocumentId.ProjectId;
```
Comment (jaredpar) [inline src/EditorFeatures/Core/Implementation/Suggestions/SuggestedAction.cs]: Suggestion: `preferredDocumnetId?.ProjectId`

Comment (jaredpar) [inline src/EditorFeatures/Core/Implementation/Suggestions/SuggestedAction.cs]: Nit: One variable uses `id` lower case and the other uses `Id` mixed case.  Can we make this consistent?

File: src/VisualStudio/Core/Def/Implementation/ProjectSystem/AbstractProject.cs [hunk 2]
```diff
@@ -87,6 +87,11 @@ internal abstract partial class AbstractProject : IVisualStudioHostProject
         /// </summary>
         private Guid _guid;
 
+        /// <summary>
+        /// string (Guid) of the _hierarchy project type
+        /// </summary>
+        private string _projectType;
```
Comment (jaredpar) [inline src/VisualStudio/Core/Def/Implementation/ProjectSystem/AbstractProject.cs]: `readonly`?

File: src/EditorFeatures/Core/Implementation/Suggestions/SuggestedAction.cs (hunk 1 above)
Comment (heejaechang) [inline src/EditorFeatures/Core/Implementation/Suggestions/SuggestedAction.cs]: will do in master.

File: src/VisualStudio/Core/Def/Implementation/ProjectSystem/AbstractProject.cs (hunk 2 above)
Comment (heejaechang) [inline src/VisualStudio/Core/Def/Implementation/ProjectSystem/AbstractProject.cs]: will do in master

**Comment by MattGertz** [comment]
Approved 5/29 by ML Shiproom.

**Pull request merged**